makedebian.pl

```diff
@@ -292,7 +298,8 @@
 noperlpath=1
 nouninstall=1
 nostart=1
-export config_dir var_dir perl autoos port login crypt host ssl nochown autothird noperlpath nouninstall nostart allow atboot makeboot
+nostop=1
```

Comment:
N: We cannot allow `setup.sh` to perform any actions on the process when run from scriptlets. It must happen once and in predictable manner.